src/JsonSchema/Enum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace JsonSchema;
+
+class Enum extends \MabeEnum\Enum
+{
```

Comment:
If that's the case, what is the motivation for this PR? There's nothing that uses it other than `ConstraintError`, and if you aren't intending users to extend it either...